react-frontend/src/reducers/loginReducer.test.js

```diff
@@ -0,0 +1,72 @@
+import expect from "expect";
+
+import reducer from "reducers/loginReducer";
+import actionGenerator from "utils/actionGenerator";
+
+const actionStatus = actionGenerator("LOGIN");
+
+describe("post reducer", () => {
+  it("should return the initial state", () => {
+    expect(reducer(undefined, {})).toEqual({
+      status: null,
+      data: { token: null },
+      error: { message: null },
+    });
+  });
+
+  it("should handle 'LOGIN_SUCCESS' action", () => {
+    expect(
+      reducer(
+        {},
+        {
+          type: actionStatus.success,
+          payload: {
+            status: 200,
+            value: { token: "token" },
+          },
+        }
```

Comment:
Follow the guidelines: https://github.com/joshsoftware/peerly/wiki/React-Development-Guidelines#directory-structure